src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -3302,6 +3302,28 @@ private void StepRunLoop_Core(bool force = false)
 				UpdateToolsAfter();
 			}
 
+			// Mutes the game if EmuHawk is unfocused or minimized
+			// An option in the sound settings needs to be turned on for this to work.
+			if (Config.MuteInBG && (WindowState == FormWindowState.Minimized || Form.ActiveForm == null))
+			{
+				Sound.StopSound();
+			}
+			else
+			{
+				Sound.StartSound();
+			}
+
+			// Mutes the game if the current FPS reaches the threshold chosen by the user or lower
+			// An option in the sound settings needs to be turned on for this to work.
+			if (Config.MuteOnLag && _lastFps <= PlatformFrameRates.GetFrameRate(Emulator.SystemId, Emulator.HasRegions() && Emulator.AsRegionable().Region is DisplayType.PAL) * ((double)Config.FPSThresholdPercentage / 100))

```

Comment:
PlatformFrameRates are not intended to query the actual core's framerate per se (they are rather "political numbers"). The core's IVideoProvider provides this kind of information and should be used.